pkgs/applications/audio/pulseaudio-dlna/0001-setup.py-remove-dbus-python-from-list.patch

```diff
@@ -0,0 +1,25 @@
+From a4bf7df795146c843696daee8c02826ba0034298 Mon Sep 17 00:00:00 2001
+From: Florian Klink <flokli@flokli.de>
+Date: Sun, 21 Nov 2021 12:04:48 +0100
+Subject: [PATCH] setup.py: remove dbus-python from list
+
+I wasn't able to convince setuptools to find this.
+---
+ setup.py | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index 61d6831..013fff3 100644
+--- a/setup.py
++++ b/setup.py
+@@ -42,7 +42,6 @@ setuptools.setup(
+     install_requires=[
+     'docopt',
+     'chardet',
+-    'dbus-python',
```

Comment:
This seems very strange, maybe someone with more knowledge on how Python works in nixpkgs can help here.

CC @SuperSandro2000 .

Reply:
I found https://github.com/PeterZainzinger/pulseaudio-dlna/commit/a8d76f18ded1f57e275cc9cb27a6843268600539 (a fork of this), so we might not be the only ones with this problem.

It really looks like setuptools doesn't properly discover it.

Reply:
Not sure, this should just work.

Reply:
But you did remove the patch for a quick test, and can confirm it doesn't work without? ;-)

Reply:
@SuperSandro2000 it seems https://github.com/NixOS/nixpkgs/pull/119469 (which you reviewed as well) had the same problem. I propose we keep this patch for now.